packages/flutter/lib/src/material/snack_bar.dart

```diff
@@ -204,6 +219,31 @@ class SnackBar extends StatefulWidget {
   /// used, if that is also null, the default value is 6.0.
   final double elevation;
 
+  /// Empty space to surround the snack bar.
+  ///
+  /// This property is only used when [behavior] is [SnackBarBehavior.floating].
+  ///
+  /// If this property is null, then the default is
+  /// `EdgeInsets.fromLTRB(15.0, 5.0, 15.0, 10.0)`.
+  final EdgeInsetsGeometry margin;
+
+  /// The amount of horizontal space to apply to the snack bar's content and
```

Comment:
it's only horizontal-only by default right? maybe update this comment to say what it will be applied as

Reply:
Oops, fixed